mdn/scrape.py

```diff
@@ -90,7 +90,7 @@
 compat_div_id = qtext
 compat_table = "<table class=\"compat-table\">" _ compat_body _ "</table>" _
 compat_body = "<tbody>" _ compat_headers _ compat_rows* _ "</tbody>"
-compat_headers = tr_open _ "<th>Feature</th>" _ compat_client_cells _ "</tr>" _
+compat_headers = tr_open _ th_open _ "Feature</th>" _ compat_client_cells _ "</tr>" _
```

Comment:
``children[0]`` is tr_open, ``children[1]`` is _ (whitespace).  ``children[2]`` used to be ``<th>Feature</th>``, but now it is th_open.  Adding that moved the index for ``compat_client_cells``, which is why the assertion is failing.